bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -402,9 +404,11 @@ export default {
     loadData (useCredentials) {
       const useCredentialsPromise = (useCredentials) ? this.setBasicCredentials() : Promise.resolve()
       return useCredentialsPromise
-        .then(() => { return this.$oh.api.get('/rest/') })
+        .then(() => { return Framework7.request.promise.json('/rest/') })
```

Comment:
I chose to use the unwrapped request promise here to get access to the error object, which is logged. This may prove really helpful for other cases where MainUI doesn't load with the cached service-worker due to some API request failure.